css/style.css

```diff
@@ -1,5 +1,168 @@
 /* General page styles */
 
-
 /* We should use a mobile first approach having out breack point at 768px */
-
+html {
+  font-family: "Roboto", sans-serif;
+  margin: 5%;
+}
+@media (min-width: 426px) {
```

Comment:
why do you set this break?
what if my screen is less than 426px? no css is applied?